### ISSUE TITLE
Subscribe to author to receive notifications about new or updated levels

### ISSUE THREAD

**Issue by Castigavi** [rr-/trcustoms#213]
It would be really cool if the site had an option to subscribe to author page in a similar fashion to Youtube. Once user subscribes to author's page, he could then receive notifications (either via e-mail or notifications tab on the site) whenever a new level is added by that author or a new version of an older level is uploaded.

**Comment by Klona13** [comment]
We can make it so that when a level is approved, an email is sent to the subscriber list.

But there's a small issue with this that severely complicates things with level updating. Authors who update a level then repeatedly keep making changes over the next few minutes and clicking on update, it would result in a lot of emails being sent, unless we limit this to new versions being uploaded only.

So the solution would be to email only if a new version is uploaded.

**Comment by rr-** [comment]
@Klona13 what if we do a daily digest? Though that does complicate things a bit. I'd rather notify only about new level files.

**Comment by Klona13** [comment]
This isn't a feature I'm too keen on honestly as I don't think we'll receive more than an average of 20 levels each month. And with launch, many users are updating their levels to fit the new standards so it's going to get a little messy with the updates until things calm down.

Plus, it's really to search for the author and sort by Last updated with #209. I think that sorting will do for now instead of overcomplicating it and filling the site with things it can already do in a simpler way in a manner that is just as effective.

**Comment by rr-** [comment]
Yeah I'm with you. I doubt we'll ever want more unless suddenly people start uploading 20 levels a day.
